yt/visualization/particle_plots.py

```diff
@@ -182,6 +186,14 @@ class ParticleProjectionPlot(PWViewerMPL):
     data_source : YTSelectionContainer Object
          Object to be used for data selection.  Defaults to a region covering
          the entire simulation.
+    deposition : string, optional
```

Comment:
Our docstrings all over the codebase reflect this convention--if it's a kwarg, it's deemed "optional." If you want that changed that will have to be addressed in a separate PR after some discussion, because it's a long-standing precedent.

Reply:
Going to implement this suggestion in any cased